lab_3_keywords_textrank/main.py

```diff
@@ -385,7 +454,13 @@ def add_edge(self, vertex1: int, vertex2: int) -> int:
                 0 if edge was added successfully, otherwise -1
         In case of vertex1 being equal to vertex2, -1 is returned as loops are prohibited
         """
-        pass
+        if vertex1 == vertex2:
+            return -1
+        for vrtx in vertex1, vertex2:
+            if vrtx not in self._edges.keys():
+                self._edges[vertex1] = self._edges.get(vertex1, []) + [vertex2]
+                self._edges[vertex2] = self._edges.get(vertex2, []) + [vertex1]
```

Comment:
do I understand it correctly: you only add the edge if at lest one vertex was not present in the graph before?